frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ProjectSelector.tsx

```diff
@@ -50,11 +57,19 @@ const ProjectSelector: React.FC<ProjectSelectorProps> = ({
               }
             }}
             value={selectedProject?.metadata.name}
-            options={projects.map((project) => ({
-              key: project.metadata.name,
-              value: project.metadata.name,
-              label: getDisplayNameFromK8sResource(project),
-            }))}
+            options={
+              isOciModel
+                ? kserveProjects.map((project) => ({
+                    key: project.metadata.name,
+                    value: project.metadata.name,
+                    label: getDisplayNameFromK8sResource(project),
+                  }))
+                : projects.map((project) => ({
+                    key: project.metadata.name,
+                    value: project.metadata.name,
+                    label: getDisplayNameFromK8sResource(project),
+                  }))
+            }
```

Comment:
Nit: this could be deduplicated.

```tsx
options={(isOciModel ? kserveProjects : projects).map((project) => ({
  key: project.metadata.name,
  value: project.metadata.name,
  label: getDisplayNameFromK8sResource(project),
}))}
```

Reply:
Thanks. Addressed.